extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -82,7 +82,10 @@ public function main() {
 			$this->wg->Out->redirect($this->title->getFullURL());
 
 			wfProfileOut( __METHOD__ );
-			return;
+
+			// SUS-1078: Don't render template when handling a POST request (block save) to prevent warnings
+			// We will be redirected back to Special:Phalanx anyways
+			return false;
```

Comment:
Can we use `skipRendering()` method here instead? I think it will make code a bit easier to read.